google-cloud-bigquery/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+### 1.8.0 / 2018-08-31
```

Comment:
Do you want to update this date at all?

Reply:
Yes, thanks for making a note of that. In the future these release PRs should be generated, reviewed, merged and released on the same day. However, @blowmage asked for a hold on this one until #2338 is merged. I'll add a do not merge label now.